scripts/DocGenerator.php

```diff
@@ -128,15 +135,20 @@ private function buildDescription($docBlock, $content = null)
         return $this->markdown->parse(implode('', $parsedContents));
     }
 
-    private function buildMethods($methods)
+    private function buildMethods($methods, $className)
     {
         $methodArray = [];
-        foreach ($methods as $method) {
+        foreach ($methods as $name => $method) {
             if ($method->getVisibility() !== 'public') {
                 continue;
             }
 
-            $access = $method->getDocBlock()->getTagsByName('access');
+            $docBlock = $method->getDocBlock();
+            if (is_null($docBlock)) {
```

Comment:
@dwsupplee same thing here. just dumping some info to find missing doc info in the library.